core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -376,6 +379,10 @@ public void close() {
     log.debug("Kafka store shut down complete");
   }
 
+  public void waitForInit() throws InterruptedException {
+    initLatch.await();
```

Comment:
nit: since this could be called multiple times, should you add a `if initLatch.getCount() > 0` condition. It'd avoid going through some Java locking machinery every time this method is called.